pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -0,0 +1,127 @@
+// Package baremetal collects bare metal specific configuration.
+package baremetal
+
+import (
+	"fmt"
+
+	"github.com/openshift/installer/pkg/ipnet"
```

Comment:
nit: for consistency we keep the imports in 3 groups
```
<stdlib>

<external>

<installer>
```

Reply:
Fixed.